pymc/logprob/mixture.py

```diff
@@ -483,3 +484,71 @@ def logprob_MixtureRV(
     "basic",
     "mixture",
 )
+
+
+class MeasurableIfElse(IfElse):
+    """Measurable subclass of IfElse operator."""
+
+
+MeasurableVariable.register(MeasurableIfElse)
+
+
+@node_rewriter([IfElse])
+def find_measurable_ifelse_mixture(fgraph, node):
+    rv_map_feature = getattr(fgraph, "preserve_rv_mappings", None)
+
+    if rv_map_feature is None:
+        return None  # pragma: no cover
+
+    if isinstance(node.op, MeasurableIfElse):
+        return None
+
+    # Check if all components are unvalued measuarable variables
+    if_var, *base_rvs = node.inputs
```

Comment:
Doesn't `base_rvs` always have length 2? Would it be better to write it as `[X_rv, Y_rv]` or something that shows the two components?

Reply:
No, `base_rvs` can have arbitrarily `n*2` length. IfElse allows multiple outputs in both branches, but they have to be the same number